tests/features/common/kie-common.feature

```diff
@@ -14,6 +11,16 @@ Feature: RHPAM and RHDM common tests
      And run sh -c '/usr/bin/rpm -q java-1.8.0-openjdk-headless || true' in container and check its output for package java-1.8.0-openjdk-headless is not installed
      And run sh -c '/usr/bin/rpm -q java-1.8.0-openjdk || true' in container and check its output for package java-1.8.0-openjdk is not installed
 
+  Scenario: Ensure the maven 8+ is installed
```

Comment:
maven 3.8.6+ ?

Reply:
just 3.8 is enough.
thanks for sppoting this.